### ISSUE TITLE
Payload may not be larger than 1048576 bytes

### ISSUE THREAD

**Issue by UnnamedWei** [Luke100000/ImmersivePaintings#82]
**Describe the bug**
Server crashed and when someone enter the server crashes again

**Latest.log**
[crash-2024-08-18_16.43.56-server.txt](https://github.com/user-attachments/files/16647832/crash-2024-08-18_16.43.56-server.txt)

**Comment by Luke100000** [comment]
Apparently you have quite a lot of paintings. For now, add https://www.curseforge.com/minecraft/mc-mods/packet-fixer

**Comment by UnnamedWei** [comment]
Oh no, what I just did is just updated the mod to 0.67 from 0.65, and rollbacked the immersive_painting.dat under world/data, will it happens again? I thought it was because some player updated a painting that is too big and it was fixed in 0.66

**Comment by Luke100000** [comment]
yes, it will happen as soon as people upload some more painting again. It's about the count, not the size here.

**Comment by UnnamedWei** [comment]
Is it about the size of that data file? Is there a way I can clean some old paintings? I tried to delete them in cache but didn’t seems working. Thank you so much

**Comment by Luke100000** [comment]
Yes, delete them via the GUI. The cache is just a cache. Or you just install the fix above, it's a general Minecraft related issue.

**Comment by UnnamedWei** [comment]
Thanks a lot! Will this fixer fix this forever? Or it was just a temporary tool? Will this oversized packet effects the server’s performance? And if it was, will you update the mod to fix this? Thanks!

**Comment by Luke100000** [comment]
Yes, it will fix it forever + any other mods with similar issues will also be fixed. Huge modpacks for example need this fix anyways. No, it won't affect server noticeably. I will fix it eventually, it's definitely on oversight on my side, tho you are the first one to stumble across it :)

**Comment by UnnamedWei** [comment]
Thanks a lot! Great work as always!

**Comment by UnnamedWei** [comment]
Hello, sorry for disturbing but seems that packet fixer can cause some bugs like chunks not being sent to player properly sometimes, just wanna ask is there a chance we can know when will be the next update of Immersive Painting to fix this? Thanks a lot!
